.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: Test
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Set common environment variables
+        run: |
+          echo "build_date=$(date -u '+%Y%m%dT%H%M%SZ')" >> $GITHUB_ENV
+          echo "build_version=$(cat VERSION)" >> $GITHUB_ENV
+
+      - name: Build Docker base image
+        run: |
+          docker pull digitalmarketplace/base
```

Comment:
Why do we need to explicitly pull this image?

Reply:
It's so in the next command we can build using `--cache-from` and potentially save some time. The Docker docs aren't very clear on this point , but Gitlab has it in [theirs](https://docs.gitlab.com/ee/ci/docker/using_docker_build.html#how-docker-caching-works) for how to speed up Docker builds on CI